action.yml

```
@@ -1,26 +1,41 @@
 ﻿# action.yml
-name: 'Install Snowflake CLI'
-description: 'Download and install snowflake-cli through pipx'
+name: "Install Snowflake CLI"
+description: "Download and install snowflake-cli through uv"
```

Is that important for users?
```suggestion
description: "Download and install snowflake-cli"
```